Tasks/DownloadBuildDropV0/task.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
mark this task as preview

Reply:
how?

Reply:
https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/AzureFileCopyV2/task.json#L19

Reply:
done